src/app/drivingLicence/controllers/root.js

```diff
@@ -1,19 +1,73 @@
 const { Controller: BaseController } = require("hmpo-form-wizard");
 
+const {
+  API: {
+    BASE_URL,
+    PATHS: { PERSON_INFO }
+  }
+} = require("../../../lib/config");
+
 class RootController extends BaseController {
   async saveValues(req, res, next) {
-    const sharedClaims = req.session?.shared_claims;
+    req.sessionModel.reset();
+    req.sessionModel.set("isAuthSourceRoute", false);
```

Comment:
Do we want to be resetting the session model we run the risk here of something failing and putting the user down a different route to the first one they were on if they navigate back